external/src/main/java/com/redhat/cloud/policies/engine/lightweight/TriggersLoader.java

```diff
@@ -47,30 +53,58 @@ public List<FullTrigger> getTriggers(String accountId) {
     @CacheResult(cacheName = "account-latest-update")
     LocalDateTime findLatestUpdate(String accountId) {
         LOGGER.debugf("Finding latest triggers update for account %s", accountId);
-        String hql = "SELECT latest FROM AccountLatestUpdate WHERE accountId = :accountId";
-        try {
-            return statelessSessionFactory.getCurrentSession().createQuery(hql, LocalDateTime.class)
-                    .setParameter("accountId", accountId)
-                    .getSingleResult();
-        } catch (NoResultException e) {
-            LOGGER.debugf("No latest triggers update found for account %s", accountId);
-            return LocalDateTime.MIN;
+
+        if (useOrgId) {
+            String hql = "SELECT latest FROM AccountLatestUpdate WHERE orgId = :accountId";
+            try {
+                return statelessSessionFactory.getCurrentSession().createQuery(hql, LocalDateTime.class)
+                        .setParameter("accountId", accountId)
+                        .getSingleResult();
+            } catch (NoResultException e) {
+                LOGGER.debugf("No latest triggers update found for account %s", accountId);
+                return LocalDateTime.MIN;
+            }
+        } else {
+            String hql = "SELECT latest FROM AccountLatestUpdate WHERE accountId = :accountId";
+            try {
+                return statelessSessionFactory.getCurrentSession().createQuery(hql, LocalDateTime.class)
+                        .setParameter("orgId", accountId)
```

Comment:
Here the 1st param of `setParameter` should be `accountId` as defined in Line 68